libs/components/filter-bar/src/lib/modules/filter-bar/models/filter-bar-filter-modal-config.ts

```diff
@@ -1,9 +1,22 @@
 import { Type } from '@angular/core';
 
+import { SkyFilterBarFilterModalSizeType } from './filter-bar-filter-modal-size';
+
 export interface SkyFilterBarFilterModalConfig {
+  /**
+   * The modal component to display when the user selects the filter.
+   * The component should expect a `SkyFilterBarFilterModalContext` object to be provided by the modal service on instantiation.
+   * The return value of the modal save action should be a `SkyFilterBarFilterValue`.
+   * @required
+   */
   modalComponent: Type<unknown>;
-
-  modalSize?: 'small' | 'medium' | 'large' | 'full';
-
+  /**
+   * The size of the modal to display.
+   * @default 'medium'
+   */
+  modalSize?: SkyFilterBarFilterModalSizeType;
+  /**
+   * Any additional properties to be passed in to the filter modal.
```

Comment:
```suggestion
   * Additional properties to pass to the filter modal. For example, on a single-filter modal with all filter types, use this property to pass in available options for each filter field.
```
I think we can tighten up and avoid passive voice, but please just ignore if we need passive voice in this case for clarity.

Also, can we provide more description of what additional properties would be passed? I think even just an example here would be helpful to explain the purpose of the property. (i.e. "Additional properties, such as x and y, to pass to the filter modal."

Reply:
The code example demonstrates this by using a single filter modal component across all the different filter types, and using this property to pass in the available options for each filter field

Reply:
I rewrote the suggestion to include a second sentence describing the example. I can rework the update as necessary; just let me know.

Reply:
I don't see this property in place anymore, so I think we can close this comment thread. Is that correct?

Reply:
Correct, this property has been functionally replaced by the `modalOpened` output